atlasdb-perf/build.gradle

```diff
@@ -7,6 +7,14 @@ apply plugin: 'application'
 mainClassName = 'com.palantir.atlasdb.performance.cli.AtlasDbPerfCli'
 applicationName = 'atlasdb-perf'
 
+configurations.all {
+    resolutionStrategy.eachDependency { DependencyResolveDetails details ->
+        if (details.requested.group == 'com.google.code.findbugs' && details.requested.name == 'annotations') {
+            details.useTarget group: 'com.google.code.findbugs', name: 'findbugs-annotations', version: '3.0.1'
+        }
+    }
+}
```

Comment:
OK, a bit nasty but it looks like these annotations are a transitive of some of the dependencies.